config/install/acquia_lift.settings.yml

```diff
@@ -1,10 +1,21 @@
-access_key: ''
-secret_key: ''
-account_name: ''
-customer_site: ''
-api_url: ''
-js_path: ''
-capture_identity: null
-vocabulary_mappings: null
-udf_mappings: null
-tracked_actions: null
+credential:
```

Comment:
A better name here would be "account" instead of "credential", it is no effort to change sine we have full test suites now. However, Burda's already using the module with hard coding their credentials, and I don't want to spend efforts on an update hook just yet.